extensions/frontend-tiles/src/integration-test/utils/webpack-config.js

```diff
@@ -0,0 +1,48 @@
+// /*---------------------------------------------------------------------------------------------
+//  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+//  * See LICENSE.md in the project root for license terms and full copyright
+//  *notice.
+//  *--------------------------------------------------------------------------------------------*/
+
+const path = require('path');
+const {globSync} = require('glob');
+const frontendLib = path.resolve(__dirname, '../../../lib/cjs');
+const Dotenv = require('dotenv-webpack');
+
+module.exports = [{
```

Comment:
since we dont have multiple configs, lets just return a single config object, not an array

Reply:
fixed in latest commit.